File: RSPAClient/Core/AtPar.POCOEntities/PAR_MNGT_VENDOR.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class PAR_MNGT_VENDOR
    {
        public string ORG_GROUP_ID { get; set; }
        public string VENDOR_ID { get; set; }
        public string VENDOR_NAME { get; set; }
        public string CONTACT_NAME { get; set; }
        public string ADDRESS1 { get; set; }
        public string ADDRESS2 { get; set; }
        public string CITY { get; set; }
        public string STATE { get; set; }
        public string COUNTRY { get; set; }
        public string ZIP { get; set; }
        public string PHONE { get; set; }
        public string FAX { get; set; }
        public string CONTACT_E_MAIL { get; set; }
        public string ORDER_DESPATCH_TYPE { get; set; }
        public Nullable<bool> STATUS { get; set; }
        public System.DateTime LAST_UPDATE_DATE { get; set; }
        public string LAST_UPDATE_USER { get; set; }
        public string LAST_CLIENT_ADDRESS { get; set; }
        public string ALLOW_VEND_ACCESS { get; set; }
        public string BILL_ONLY_EMAIL { get; set; }
        public short REMINDER_FREQ { get; set; }
        public string VEND_USER_ID { get; set; }
        public string ADD_ITEMS_LFLAG { get; set; }
        //Ignore Properties
        public string ORG_GROUP_NAME { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/PAR_MNGT_VENDOR.ts
/// <reference path="atpar_validation_rules.ts" />
import { ATPAR_VALIDATION_RULES } from "./atpar_validation_rules";
export class PAR_MNGT_VENDOR {
    public ORG_GROUP_ID: string;
    public VENDOR_ID: string;
    public VENDOR_NAME: string;
    public CONTACT_NAME: string;
    public ADDRESS1: string;
    public ADDRESS2: string;
    public CITY: string;
    public STATE: string;
    public COUNTRY: string;
    public ZIP: string;
    public PHONE: string;
    public FAX: string;
    public CONTACT_E_MAIL: string;
    public ORDER_DESPATCH_TYPE: string;
    public STATUS?: boolean;
    public LAST_UPDATE_DATE:  Date;
    public LAST_UPDATE_USER: string;
    public LAST_CLIENT_ADDRESS: string;
    public ALLOW_VEND_ACCESS: any;
    public BILL_ONLY_EMAIL: string;
    public REMINDER_FREQ: number;
    public VEND_USER_ID: string;
    public ADD_ITEMS_LFLAG: string;
    public MODE: string = "ADD";
    public validationRules: ATPAR_VALIDATION_RULES[];
}